samples/matter/contact_sensor/sysbuild/mcuboot/boards/nrf5340dk_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_NORDIC_QSPI_NOR=y
+CONFIG_NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE=4096
+CONFIG_NORDIC_QSPI_NOR_STACK_WRITE_BUFFER_SIZE=16
+
+# The following configurations are required to support simultaneous multi image update
+CONFIG_PCD_APP=y
+CONFIG_UPDATEABLE_IMAGE_NUMBER=2
+
+CONFIG_BOOT_SWAP_USING_MOVE=n
+# Multi-image updates do not support image swapping yet.
+CONFIG_BOOT_UPGRADE_ONLY=y
+
+# The network core cannot access external flash directly. The flash simulator must be used to
+# provide a memory region that is used to forward the new firmware to the network core.
+CONFIG_FLASH_SIMULATOR=y
+CONFIG_FLASH_SIMULATOR_DOUBLE_WRITES=y
+CONFIG_FLASH_SIMULATOR_STATS=n
+
```

Comment:
```suggestion
```

Reply:
Do you mean it is not needed anymore? I can see that the .config has these options configured correctly, even if I remove this lines. Where are these configs configured for nRF5340?

Reply:
Ok, I think it's partially handled by `SB_CONFIG_MCUBOOT_NRF53_MULTI_IMAGE_UPDATE`